randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -3100,7 +3102,7 @@ Extra - asset_id: collision_camera_040
   * Extra - actor_def: actordef:actors/props/dooremmy/charclasses/dooremmy.bmsad
   > Dock to Purple EMMI Introduction Access
       All of the following:
-          Morph Ball and After Ferenia - Central Unit
+          Morph Ball
```

Comment:
this is an example of one that *does* need the CU event. You don't need to do anything regarding the CU event. It should all be correct as is.